triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/process/wrapper/TriggerTimer.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger.simulation.process
+package wrapper
+
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.actor.typed.scaladsl.Behaviors
+import com.daml.lf.engine.trigger.TriggerMsg
+
+import scala.concurrent.duration.FiniteDuration
+
+object TriggerTimer {
```

Comment:
`TriggerTimer` allows a more flexible scheduling of heartbeat messages. The obvious generalisation here is to allow messages other than heartbeats to be scheduled.